docs/index.md

```diff
@@ -30,90 +30,73 @@ The following 3 license types (`jq .type`) do **NOT** support APIs:
 - OSS
 
 ## Example Usage
-```hcl
+
+```terraform
```

Comment:
hcl format isn't a thing?

Reply:
I think both are supported. The plugin uses `terraform` 🤷‍♂️